endpoints/api/ai.py

```
@@ -0,0 +1,585 @@
+        allowed, error_message = check_ai_subscription_required(owner)
+        if not allowed:
+            raise InvalidRequest(error_message)
+
+        # In managed mode, AI is always available for paid subscribers
+        # In BYOK mode, check if the org has enabled and configured AI
+        if not is_managed_mode():
+            if not is_description_generator_enabled(owner):
+                raise InvalidRequest(
+                    "AI description generation is not enabled for this organization. "
+                    "Please configure AI settings first."
+                )
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)